nextjs-app-v13/src/pages/components/lottie-player.tsx

```diff
@@ -0,0 +1,32 @@
+import NavigationLayout from "@/layout/navigation";
+import { PieLottiePlayer } from '@justeattakeaway/pie-lottie-player/dist/react.js';
+import { PieButton } from '@justeattakeaway/pie-button/dist/react.js';
+import { useState } from "react";
+import { isServer } from 'lit';
+
+const animations = [
+    "/animations/preparing.json",
+    "/animations/courier.json",
```

Comment:
On the Next 13 app when I press the "load another animation" button, it loads the courier animation but it doesn't play, same again for the order confirmed animation.

Reply:
Doing this in the console manages to get it playing
`document.querySelector('pie-lottie-player').play()`
so it is loading correctly, just not autoplaying for some reason

Reply:
Good catch!

Reply:
Fixed it here: https://github.com/justeattakeaway/pie/pull/1845